jobbergate-cli/jobbergate_cli/subapps/job_scripts/app.py

```diff
@@ -112,6 +116,9 @@ def get_one(
 @app.command()
 def create_stand_alone(
     ctx: typer.Context,
+    job_script_path: pathlib.Path = typer.Argument(
```

Comment:
Do we need to support this as an option as well so we don't break existing workflows?

Reply:
Nice catch. Addressed by aa2b40ed7c43125bc91edc8ed1b4b6e2cfc443ff.